### PR TITLE
Turn-exercise-061A-into-practice

### LINKED ISSUE #2246: Turn exercises 061A to 068 into practice
This task consists of doing 10 exercises instead of one, as described in issues #2167, #2168, #2170, #2172, #2173, #2176, #2177, #2178, #2179 and #2180. Create 10 folders, and populate them as the others. Makes sure `dune build --root .` and `dune test --root . answer` work.

- [X] 061A #2259
- [X] 061B #2260
- [X] 062A #2261
- [X] 062B #2262
- [X] 063 #2263
- [X] 064 #2264
- [X] 065 #2265
- [X] 066 #2266
- [X] 067 #2267
- [X] 068 #2268

### PR DESCRIPTION
**Pull request by Appleeyes**
This PR is referencing the issue https://github.com/ocaml/ocaml.org/issues/2246

This is part of the issue, I decided to do different PR for each folder so it wont be too much work in a PR
@cuihtlauac please review

### CODE REVIEW

File: practice/061A/answer/impl.ml
Comment (cuihtlauac) [inline practice/061A/answer/impl.ml:1]: Does not compile, make sure `dune build --root .` works.

Your need to copy the type declaration in this file

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Also fix newline at eof

**Comment by Appleeyes** [comment]
> Also fix newline at eof

@cuihtlauac I always make sure all the files have new line at eof, it doesnt quite show on github but it is present once you pull to your editor or view elsewhere

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Use the tree from exercice 62B to have more meaningful tests

**Review by cuihtlauac — APPROVED** [review]
Thanks @Appleeyes

**Pull request merged**